lang/cs/Org.Apache.REEF.Examples.HelloREEF/HelloREEF.cs

```diff
@@ -74,6 +76,8 @@ private static IConfiguration GetRuntimeConfiguration(string name)
                     return LocalRuntimeClientConfiguration.ConfigurationModule
                         .Set(LocalRuntimeClientConfiguration.NumberOfEvaluators, "2")
                         .Build();
+                case YARN:
+                    return YARNClientConfiguration.ConfigurationModule.Build();
```

Comment:
That parameter has no correspondent in YARN. The number of Evaluators set for the local runtime defines the size of the cluster simulated by it, not the number of Evaluators the job uses.